desktop/src/main/java/bisq/desktop/main/overlays/windows/ContractWindow.java

```diff
@@ -265,9 +265,6 @@ private void addContent() {
         addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.makerFeeTxId"), offer.getOfferFeePaymentTxId());
         addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.takerFeeTxId"), "TAKER FEE TX ID NOT PART OF CONTRACT");  // TODO (woodser): should taker fee tx id be part of contract?
 
-        if (dispute.getDepositTxSerialized() != null)
```

Comment:
This removes the deposit tx hash from the contract window. Per our conversation, this view was to show the maker and taker deposit tx hashes.

Reply:
Ah I misunderstood. Thought there's going to be a follow up PR. Please check again.